src/Symfony/Bundle/FrameworkBundle/Resources/config/form.xml

```diff
@@ -7,6 +7,7 @@
     <services>
         <!-- ResolvedFormTypeFactory -->
         <service id="form.resolved_type_factory" class="Symfony\Component\Form\ResolvedFormTypeFactory" />
+        <service id="Symfony\Component\Form\ResolvedFormTypeFactoryInterface" alias="form.resolved_type_factory" public="false" />
```

Comment:
Is it actually needed ? I never had a need to inject it directly. This service should probably only be used by the form registry and the form factory, not by userland code